database/seeders/AccessibilitiesSeeder.php

```
@@ -0,0 +1,1489 @@
+use Illuminate\Support\Facades\DB;
+
+class AccessibilitiesSeeder extends Seeder
+{
+    /**
+     * Run the database seeds.
+     *
+     * @return void
+     */
+    public function run()
+    {
+        $acccessibilites_json = '
```

PHPMD (2.15.0)

The variable $acccessibilites_json is not named in camelCase. (Controversial Rules)

(CamelCaseVariableName)